Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md

```diff
@@ -85,21 +85,37 @@ Ensure you read the [overview](index.md) before you begin - you will need to ens
 * You will need to setup 2 x Azure Web Apps - one for the master (administrative) environment and another for your front-end environment
 * You will need 1 x SQL server that is shared with these 2 web apps
 
-###Lucene/Examine configuration
+### Lucene/Examine configuration
 
-* Ensure you are using the [latest Examine version from Nuget](https://www.nuget.org/packages/Examine)
-* In ExamineSettings.config, you can add these properties to all of your indexers and searchers: 
-```
-useTempStorage="Sync"
-tempStorageDirectory="UmbracoExamine.LocalStorage.AzureLocalStorageDirectory, UmbracoExamine"
-```
+#### Examine v0.1.80+ ####
+
+Examine v0.1.80 introduced a new `directoryFactory` which should be added to all indexers
+
+    directoryFactory="Examine.LuceneEngine.Directories.SyncTempEnvDirectoryFactory,Examine"
+
+The `SyncTempEnvDirectoryFactory` enables Examine to sync indexes between the remote file system and the local environment temporary storage directory, the indexes will be accessed from the temporary storage directory.
```

Comment:
Maybe use the copy about the `Sync` setting below so that it's clear what this is doing since it's the same principle about Lucene not liking working on remote file systems.